tests/search_intentions/test_top_families.py

```diff
@@ -31,6 +31,7 @@
         search_terms="egypt national climate change strategy",
         expected_family_slugs=["egypt-national-climate-change-strategy-nccs-2050_d3b1"],
         description="Searching for the family title should return the correct family.",
+        known_failure=True,
```

Comment:
when did this start failing? do we know how/why?

Reply:
I don't remember - there was a bit of confusion around this! weirdly, this doesn't fail on prod 🤔 
<img width="729" alt="image" src="https://github.com/user-attachments/assets/d1bc6833-d06f-4724-9caa-aa99ad0861b4" />

I'll dig into this

Reply:
This seems to be an issue with the app, or known behaviour which needs to be adapted into these search tests - [Slack message](https://climate-policy-radar.slack.com/archives/C078UT6R8GK/p1740394509870239) here. Closing this for now as we'll pick this up elsewhere.